### PR TITLE
fix: refactored some styles in jinkie player and removed br tag - #865

### LINKED ISSUE #865: song progress bar with mozilla firefox not aligned properly
The seek (song progress bar) is touching the bottom of page while using firefox but works fine with google chrome

screenshot

### PR DESCRIPTION
**Pull request by armedev**
fixes #865

This pull request has some styles overwritten for our use case in react-jinkie-player, It also removes the br tag in audioTitle, and some additional style to keep the layout as before.

### CODE REVIEW

**Comment by armedev** [comment]
@deluan this PR is ready to be merged I checked it on all major browsers font-size options, now the layout doesn't go out of view.
also, I have moved some styles (`& .play-mode-title` ) misplaced by this PR(#1041)

File: ui/src/audioplayer/Player.js [hunk 1]
```diff
@@ -48,13 +51,17 @@ const useStyle = makeStyles(
...
     player: {
       display: (props) => (props.visible ? 'block' : 'none'),
-    },
-    artistAlbum: {
-      marginTop: '2px',
+      '& .progress-bar-content': {
+        display: 'flex',
+        flexDirection: 'column',
+      },
       '& .play-mode-title': {
```
Comment (armedev) [inline ui/src/audioplayer/Player.js:58]: This block of code should be inside `player` to work.

Comment (deluan) [reply ui/src/audioplayer/Player.js:58]: Yes, this was the result of a bad merge. Thanks for fixing it.

**Review by deluan — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**